Orange/widgets/gui.py

```diff
@@ -787,6 +787,7 @@ def __textEdited(self):
         self.__changed = True
 
     def returnPressedHandler(self):
+        self.clearFocus()
```

Comment:
No, this is usually not how you edit settings. If you change a value in one line edit, it doesn't mean you're going to continue with the next.